src/control/cmd/daos_server/README.md

```diff
@@ -7,13 +7,88 @@ tool](/src/control/cmd/dmg/README.md) and manage bring-up of the DAOS system.
 A DAOS Server instance will typically be run on every storage node to
 orchestrate the operation of the managed data plane processes.
 
+## Network and Storage Management
+
+The DAOS data plane utilises two forms of non-volatile storage,
+storage class memory (SCM) in the form of persistent memory modules
+and NVMe in the form of high-performance SSDs.
+
+The DAOS Server provides capability to provision and manage network
+devices and non-volatile storage including the allocation of
+resources to data plane instances.
+
+See
+[admin guide](https://daos-stack.github.io/admin/deployment/#hardware-provisioning)
+for details on hardware provisioning.
+
 ## Configuration
 
 A server configuration file which is passed on invokation contains details of
 global and per-io-server parameters related to hardware devices and
 environments.
 
-## Certificate
+### Configuration File
+
+A populated `daos_server` config file is required when starting
+`daos_server` process, it's location can be specified on the
+commandline (`-o` option) or default location
+(`<daos install dir>/install/etc/daos_server.yml`).
+
+Example config files can be found in the
+[examples folder](https://github.com/daos-stack/daos/tree/master/utils/config/examples).
```

Comment:
To link to internal sections, I have been using `/` to indicate the root of the GH repo, which appears to work. So this URL could be simplified to: `/utils/config/examples` for example. I think using that format also allows branches to link within the branch, rather than linking back to master.